build.ps1

```diff
@@ -1,33 +1,177 @@
 ## Copyright (c) Microsoft Corporation.
 ## Licensed under the MIT License.
-
-[CmdletBinding()]
+# default behavior is build only
+[CmdletBinding(SupportsShouldProcess=$true)]
 param (
     [Parameter()]
     [string]
-    $Configuration = "Debug",
+    $Configuration = "Release",
```

Comment:
Why default to release?  If used locally, it would make sense to build debug by default, but publishing can explicitly specify release?